docs/azure/sdk/authentication/create-token-credentials-from-configuration.md

```diff
@@ -68,39 +68,41 @@ You can create both user-assigned and system-assigned managed identities using c
 
 #### User-assigned managed identities
 
-A user-assigned managed identity can be used by providing a client ID, resource ID, or object ID:
+A user-assigned managed identity can be used by providing a client ID, resource ID, or object ID.
 
-- **Client ID:**
+## [Client ID](#tab/client-id)
 
-    ```json
-    {
-        "credential": "managedidentity",
-        "clientId":  "<clientId>"
-    }
-    ```
+```json
+{
+    "credential": "managedidentity",
+    "clientId":  "<clientId>"
+}
+```
 
-- **Resource ID:**
+## [Resource ID](#tab/resource-id)
 
-    ```json
-    {
-        "credential": "managedidentity",
-        "managedIdentityResourceId":  "<managedIdentityResourceId>"
-    }
-    ```
+```json
+{
+    "credential": "managedidentity",
+    "managedIdentityResourceId":  "<managedIdentityResourceId>"
+}
+```
 
-    The resource ID takes the form `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}`.
+The resource ID takes the form `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}`.
```

Comment:
```suggestion
The resource ID takes the form: 
`/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}`
```

Just a formatting suggestion since this gets split onto the next line anyways.